src/observation.js

```diff
@@ -0,0 +1,78 @@
+import Promise from 'bluebird'
+import isFunction from '101/is-function'
+
+class Observation {
+  constructor (name, experiment, fn) {
+    this.name = name
+    this.experiment = experiment
+    this.now = Date.now()
+    this.fn = fn
+  }
+
+  static create (name, experiment, fn) {
+    const observation = new Observation(name, experiment, fn)
+    return Promise.resolve()
+      .then((_) => (observation.fn()))
```

Comment:
interesting you bring this up. this is an artifact of how I wrote this and then upgraded `standard`. [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) says functions with no arguments need either `()` or `_` before the arrow. Until recently, `standard` didn't care, but now they want empty parens rather than an underscore. I removed them just now, but that's the logic